docs/themes/thxvscode/assets/js/package.json

```diff
@@ -3,14 +3,14 @@
 		"build": "webpack --mode=production"
 	},
 	"devDependencies": {
```

Comment:
Add `"packageManager": "yarn@1.22.22"` to this file and yarn should work as expected.

Reply:
That did it!  Added in latest.